examples/working_with_sofa_files.ipynb

```diff
@@ -0,0 +1,318 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import matplotlib as mpl\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# select matplotlib backend for plotting\n",
+    "%matplotlib inline\n",
+    "\n",
+    "# load head-related impulse responses\n",
+    "hrirs, source_coordinates, receiver_coordinates = pf.io.read_sofa(\n",
+    "    os.path.join('data', 'head_related_impulse_responses.sofa'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SOFA file used in this example is contained head-realated impulse responses (HRIRs) from the [FABIAN database](https://depositonce.tu-berlin.de/handle/11303/6153.5>). Lets find the HRIR for the source position at the left ear on the horizontal plane. It has an azimuth angle of 90 degrees and an elevation of 0 degrees"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# coordinate point to find\n",
+    "find = pf.Coordinates().from_spherical_elevation(np.pi/2, 0, 1.7)\n",
+    "# index of corresponding point\n",
+    "lateral, _ = source_coordinates.find_nearest(find, k=1)\n",
+    "\n",
+    "print(lateral)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tells us that the HRIR for the desired source position is at index 134 in `hrirs` audio object. Note that you get more then the most closest point by using different values for `k`. We can use the `Coordinates.show` method to get visual feedback for checking if we got the correct source"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_coordinates.show(lateral)\n",
+    "plt.gca().view_init(azim=-10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets also find all sources on the horizontal plane, i.e., all sources that have an elevation of 0 degree. Lets get visual feedback right away. This time we don't get an index but a logical array with is `True` at the positions where `source_coordinates.elevation` is zero."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "horizontal = source_coordinates.elevation == 0\n",
+    "source_coordinates.show(horizontal)\n",
+    "plt.gca().view_init(azim=-10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For completeness, lets also find all sources in the median plane. This can be done in a similar way, considering that they have a lateral angle of 0 degree"
```

Comment:
```suggestion
    "For completeness, let's also find all sources in the median plane. This can be done in a similar way, considering that they have a lateral angle of 0 degree"
```

Reply:
yes